cloud_blobstore/__init__.py

```diff
@@ -185,6 +199,22 @@ def get_copy_token(
         """
         raise NotImplementedError()
 
+    def get_last_modified_date(
+            self,
+            bucket: str = None,
+            key: str = None,
+            blob_obj: typing.Any = None
+    ) -> datetime:
+        """
+        Retrieves last modified date for a given key in a given bucket.
+        Alternatively, the blob object itself can be passed as a parameter.
+        :param bucket: the bucket the object resides in.
+        :param key: the key of the object for which the last modified date is being retrieved.
```

Comment:
ummm, is it from the key or blob_obj?  very confusing.

Reply:
The intention was to provide the option to supply either the bucket and key or a blob_obj. I agree the documentation could be clearer, but it's probably best if there were 2 separate functions.